rotation/node.go

```diff
@@ -61,22 +61,49 @@ func RotateNodeCredentials(
 	}
 
 	// First we get our current node information and decrypt the fetch request
-	currentNodeInfo, err := types.LoadNodeInformation(ctx, storage, currentKeyId, opt...)
-	if err != nil {
-		err := fmt.Errorf("error loading current node information: %w", err)
-		opts.WithLogger.Error(err.Error(), "op", op)
-		return nil, fmt.Errorf("(%s) %s", op, err.Error())
+	var i interface{} = storage
+	nodeIdStorage, ok := i.(nodeenrollment.NodeIdLoader)
```

Comment:
We don't need the intermediate variable, is there a reason you want to use it?
```suggestion
	nodeIdStorage, ok := storage.(nodeenrollment.NodeIdLoader)
```

Reply:
thanks!